ext/snmp/snmp.c

```diff
@@ -1374,6 +1374,28 @@ PHP_FUNCTION(snmp_set_oid_output_format)
 }
 /* }}} */
 
+/* {{{ Set the string output format. */
+PHP_FUNCTION(snmp_set_string_output_format)
+{
+	zend_long a1;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "l", &a1) == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	switch((int) a1) {
+		case NETSNMP_STRING_OUTPUT_GUESS:
+		case NETSNMP_STRING_OUTPUT_ASCII:
+		case NETSNMP_STRING_OUTPUT_HEX:
+			netsnmp_ds_set_int(NETSNMP_DS_LIBRARY_ID, NETSNMP_DS_LIB_STRING_OUTPUT_FORMAT, a1);
+			RETURN_TRUE;
```

Comment:
This return value doesn't make much sense to me. I think it would be better to make this a void function.

Reply:
See my comment about `snmp_set_oid_value_format()` above.

Reply:
I agree, shall we also modify the snmp_set_oid_value_format()?